packages/manager/apps/okms/src/modules/secret-manager/pages/secret/versionList/VersionCells.component.spec.tsx

```diff
@@ -71,13 +72,11 @@ describe('VersionCellId test suite', () => {
         // GIVEN version
 
         // WHEN
-        const { container } = await renderVersionLink(version);
+        await renderVersionLink(version);
 
         // THEN
-        await getOdsButtonByLabel({
-          container,
-          label: version.id.toString(),
-          isLink: true,
+        await getOdsLinkByTestId({
```

Comment:
I might be missing something, but I'm concerned this reduces the test's effectiveness. With this change, the test would pass even if someone puts arbitrary content in the label instead of the version number.

While this specific test isn't critical, I'd prefer not to generalize this pattern of replacing label checks with test IDs across the codebase.